doc/jetty-config.md

```diff
@@ -129,7 +129,8 @@ virtual cores on the host divided by 8, with a minimum of 1 and maximum of 4.
 This sets the number of threads that the webserver will dedicate to processing
 events on connected sockets for _encrypted_ HTTPS traffic.  Defaults to the
 number of virtual cores on the host divided by 2, with a minimum of 1 and
-maximum of 4.
+maximum of 4. The number of selectors used by Jetty is twice the number of ssl
```

Comment:
Whatever language we end up with for this, it would be good to do the same for the `selector-threads` option - which is the same as this option but for the plaintext port.